Notify/NotificationService.swift

```diff
@@ -78,52 +79,60 @@ class NotificationService: UNNotificationServiceExtension {
                             }
                         }
                         
-                        self.bestAttemptMutex.lock {
-                            
-                            let content = UNMutableNotificationContent()
-                            let (noteSubtitle, noteBody) = unsealedRequest.notificationDetails()
-                            
-                            content.subtitle = noteSubtitle
-                            content.body = noteBody
-
-                            // special case: me request
-                            if case .me = unsealedRequest.body {
-                                content.title = "\(session.pairing.displayName)."
-                            }
+                        self.bestAttemptMutex.lock()
```

Comment:
why the switch from `.lock{ }` to `.lock()`?

Reply:
rewrote mutex logic, check the latest commit